### PR TITLE
Update pluginsync to 0.1.9

### PR DESCRIPTION
**Pull request by nanliu**
* add new Snap plugins
* add documentation for plugin documentation generation
* disable download link for mesos (we don't build them)
* unmanaged files are no longer listed in .pluginsync.yml

### CODE REVIEW

**Comment by nanliu** [comment]
This should fix #35

File: README.md [hunk 1]
```diff
@@ -65,6 +65,22 @@ Please add the following to your .travis.yml file:
 
 NOTE: travis secrets are encrypted per repo. see [travis documentation](https://docs.travis-ci.com/user/encryption-keys/) for more info.
 
+Generate plugin documentation:
+1. Fork snap and add your repo to modulesync.yml
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md [hunk 2]
```diff
@@ -65,6 +65,22 @@ Please add the following to your .travis.yml file:
...
+```
+plugin_catalog.md:
+  fork: username/snap
+```
+2. Review new catalog/wishlist (optional)
+```
+$ bundle exec rake plugin:catalog
+$ bundle exec rake plugin:wishlist
+```
+3. Create the pull-request against the snap repo:
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: lib/pluginsync/github.rb [hunk 3]
```diff
@@ -178,7 +178,7 @@ def metadata
 
         metadata = yml_content('metadata.yml')
 
-        if @owner == Pluginsync::Github::INTEL_ORG
+        if (@owner == Pluginsync::Github::INTEL_ORG) and (plugin_name != 'Mesos')
```
Comment (kindermoumoute) [inline lib/pluginsync/github.rb]: Why we just don't keep in on the list `managed_modules.yml`?

Comment (nanliu) [inline lib/pluginsync/github.rb]: You mean the list of plugins with exceptions?

File: README.md [hunk 4]
```diff
@@ -65,6 +65,29 @@ Please add the following to your .travis.yml file:
 
 NOTE: travis secrets are encrypted per repo. see [travis documentation](https://docs.travis-ci.com/user/encryption-keys/) for more info.
 
+Generate plugin documentation:
+1. Generate github api token and populate [`${HOME}/.netrc` config](https://github.com/octokit/octokit.rb#using-a-netrc-file)
```
Comment (kindermoumoute) [inline README.md]: Add link: `[github api token](https://github.com/settings/tokens)`.

File: README.md [hunk 5]
```diff
@@ -65,6 +65,29 @@ Please add the following to your .travis.yml file:
...
+```
+machine api.github.com
+  login defunkt
```
Comment (kindermoumoute) [inline README.md]: Maybe clarify:
```
machine api.github.com
  login <github ID>
  password <github API token>
```

File: README.md [hunk 6]
```diff
@@ -65,6 +65,37 @@ Please add the following to your .travis.yml file:
...
+1. Generate [github api token](https://github.com/settings/tokens) and populate [`${HOME}/.netrc` config](https://github.com/octokit/octokit.rb#using-a-netrc-file)
+
+    ```
+    machine api.github.com
+      login <username>
+      password <github_api_token>
+    ```
+    NOTE: You only need to grant public repo read permission for the API token.
+
+2. Fork [Snap](https://github.com/intelsdi-x/snap) repo on github and add your repo to modulesync.yml
```
Comment (kindermoumoute) [inline README.md]: > and add your repo to modulesync.yml

Precise this file is in the pluginsync repo.

File: moduleroot/CONTRIBUTING.md [hunk 7]
```diff
@@ -39,7 +39,7 @@ $ git checkout -b some-topic-branch
 * Open a pull request for the appropriate project.
 * Contributors will review your pull request, suggest changes, and merge it when it’s ready and/or offer feedback.
 
-If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/README.md#maintainers).
+If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/CONTRIBUTING.md).
```
Comment (IzabellaRaulin) [inline moduleroot/CONTRIBUTING.md]: @nanliu, what do you think about pointing to `https://github.com/intelsdi-x/snap/blob/master/docs/MAINTAINERS.md`. In CONTRIBUTING.md there is no information how to contact to Snap's maintainers, so I suppose docs/MAINTAINERS.md should be linked instead.

File: managed_modules.yml [hunk 8]
```diff
@@ -36,18 +36,22 @@
 - snap-plugin-collector-psutil
```
Comment (kindermoumoute) [inline managed_modules.yml]: line 29, we no longer maintain node-manager plugin.

**Pull request merged**